src/config.rs

```diff
@@ -223,5 +243,18 @@ mod test {
         clean_up(&test_root);
     }
 
-
+    #[test]
+    fn can_get_inhabitable_shells() {
+        let mut test_root = set_up("inhabit-shells", 
+                                   "default", 
+                                   vec!["default", "bcd", "abc", "cde"]);
+        let config = FsConfig::new(test_root.clone());
+        let res = config.get_shell_list(); 
+        
+        assert_eq!(res[0], "abc");
+        assert_eq!(res[1], "bcd");
+        assert_eq!(res[2], "cde");
+        assert_eq!(res[3], "default");
+        clean_up(&test_root);
+    }
```

Comment:
Solid test @houngj.  I'd also like to see a test that demonstrates the behavior when there are no shell directories set up.